OpenLaTeXStudio/EditorModule/src/latexstudio/editor/menu/OpenTemplate.java

```diff
@@ -0,0 +1,121 @@
+package latexstudio.editor.menu;
+
+import java.awt.BorderLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.swing.DefaultListModel;
+import javax.swing.JDialog;
+import org.w3c.dom.Document;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import latexstudio.editor.EditorTopComponent;
+import latexstudio.editor.TopComponentFactory;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.NbBundle;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Text;
+import org.xml.sax.SAXException;
+
+@ActionID(
+        category = "File",
+        id = "latexstudio.editor.OpenTemplate"
+)
+@ActionRegistration(
+        displayName = "#CTL_OpenTemplate"
+)
+@ActionReference(path = "Menu/File", position = 1201)
+@NbBundle.Messages("CTL_OpenTemplate=Open Template")
+
+public final class OpenTemplate extends JDialog implements ActionListener {
+
+    private static final String pathToTemplatesDirectory = "/latexstudio/editor/resources/templates/";
```

Comment:
Can we move that to ApplicationUtils class? (that's where we keep all required file/directory paths)